pkg/hooks/hooks.go

```diff
@@ -98,4 +97,27 @@ func readHooks(hooksPath string, hooks map[string]HookParams) error {
 	}
 	return nil
 }
-*/
+
+var hooksMap map[string]HookParams
```

Comment:
Can we store this in the runtime instead of as a global?

Reply:
Much better idea.